TicTacToe/src/com/ticTacToe/GridService.java

```diff
@@ -0,0 +1,112 @@
+package com.ticTacToe;
+
+import com.ticTacToe.model.Player;
+import com.ticTacToe.model.TicTacToeGrid;
+
+import java.util.List;
+
+public class GridService {
+    private String[][] grid;
+    private int gridRows;
+    private int gridCols;
```

Comment:
Is this required? If we already have the 2d array, we can get it whenever required